<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>354</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>458</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>458</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;По умолчанию все сервера в Tas-IX.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/img/img/play.png);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>450</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>450</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>130</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>130</height>
       </size>
      </property>
      <property name="title">
       <string>Сейчас играет:</string>
      </property>
      <widget class="QLabel" name="playingNow">
       <property name="geometry">
        <rect>
         <x>175</x>
         <y>20</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="Phonon::VolumeSlider" name="volumeSlider">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>110</width>
         <height>100</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="pageStep">
        <number>50</number>
       </property>
      </widget>
      <widget class="QPushButton" name="playButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>458</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
